src/main/java/ch/njol/skript/effects/EffBrewingConsume.java

```diff
@@ -0,0 +1,63 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.EventRestrictedSyntax;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.event.Event;
+import org.bukkit.event.inventory.BrewingStandFuelEvent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Consume Brewing Fuel")
+@Description({
+	"Makes the brewing stand in the brewing fuel event consume its fuel.",
+	"By making it not consume the fuel, it will keep the fuel item and still add to the fuel level of the brewing stand."
+})
+@Examples({
+	"on brewing fuel consumption:",
+		"prevent the brewing stand from consuming the fuel"
+})
+@Since("INSERT VERSION")
+public class EffBrewingConsume extends Effect implements EventRestrictedSyntax {
+
+	static {
+		Skript.registerEffect(EffBrewingConsume.class,
+			"make [the] brewing stand consume [the] fuel",
+			"prevent [the] brewing stand from consuming [the] fuel");
```

Comment:
```suggestion
			"make [the] brewing stand consume [the] fuel",
			"prevent [the] brewing stand from consuming [the] fuel");
```
any thoughts on options such as `allow`, `disallow`, `block`, `stop`? not saying you need to add them, just wondering if any others think it sounds good.